src/kirby/components/floating-action-button/floating-action-button.component.tns.ts

```diff
@@ -22,6 +22,7 @@ export class FloatingActionButtonComponent extends ContentView {
   // TODO: showShadow should become an "elevation" enum in the future;
   @Input() showShadow?: boolean = true;
   @Input() disabled?: boolean = false;
+  @Input() size?: 'sm' | 'md' | 'lg' = 'sm';
```

Comment:
We normally use the short versions: 's', 'm' and 'l'

Reply:
not according to src/kirby/directives/size/size.directive.ts